src/main.rs

```diff
@@ -1,6 +1,7 @@
+use std::ffi::OsStr;
```

Comment:
Looks like these two imports are no longer used in the current revision.

Reply:
Fixed.